wgpu/backends/wgpu_native/_helpers.py

```diff
@@ -389,17 +391,18 @@ def get_dict(self):
             total = 0
             for name in names:
                 d = native_report[backend][name]
-                total += d["occupied"] + d["vacant"] + d["error"]
+                total += d["allocated"] + d["kept"] + d["released"] + d["error"]
```

Comment:
Adding these results in counts that are twice too high. I'm trying to get more insight into what best to do here https://github.com/gfx-rs/wgpu-native/issues/367

Reply:
I tried to find related PRs in the wgpu-core diff but didn't dig deep enough.

Reply:
From what I can see, the *occupied* and *vacant* simply map to *kept* and *released*. But when I tried to only remove the *allocated* field (thus making things as they were), the memtests still fail because the *released* field is nonzero. So it looks like wgpu-core keeps objects around longer for re-use. Adding only *kept* and *error* fixes things.

Well, except for some cases that were special, and are now less-special, so that's good. See the commit I just pushed.